source/nodejs/adaptivecards-designer/src/card-designer.ts

```diff
@@ -1473,6 +1473,15 @@ export class CardDesigner extends Designer.DesignContext {
 
             this.setCardPayload(card, false);
             this.updateToolbar();
+
+            const undoSuccessPopup = document.createElement("p");
+            undoSuccessPopup.textContent = "Successfully undid your last change";
+            undoSuccessPopup.className = "popup-message";
+            undoSuccessPopup.setAttribute("role", "alert");
+
+            document.body.appendChild(undoSuccessPopup);
+
+            setTimeout(() => { undoSuccessPopup.remove(); }, 3000);
```

Comment:
Is there accessiblity practice to state how long should a popup shows?

Reply:
I couldn't find a best practice on this. 

With the shortened text, 3 seconds is long enough for Narrator to fully read it before the popup is dismissed.